ui/raidboss/data/triggers/o5s.js

```diff
@@ -36,7 +36,7 @@
     {
       locale: 'fr',
       replaceSync: {
-        'Engage!': "À l'attaque!",
+        'Engage!': "À l'attaque !",
```

Comment:
No space here before the exclamation point for French engage, here and elsewhere.  See comments in #57 for more details.